### PR TITLE
DOCSP-26711: Update compat tables for PHPC and PHPLIB 1.15

### PR DESCRIPTION
**Pull request by jmikola**
# Pull Request Info

JIRA - https://jira.mongodb.org/browse/DOCSP-26711

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?

### CODE REVIEW

File: source/includes/language-compatibility-table-php.rst [hunk 1]
```diff
@@ -13,8 +14,19 @@
      - PHP 7.0
      - PHP 5.6
      - PHP 5.5
-
+   * - ext + lib 1.15
```
Comment (mongoKart) [inline source/includes/language-compatibility-table-php.rst:17]: Issue: We should probably keep a footnote similar to the current one to explain that 1) we skipped lib 1.14 and 2) the versions are now equivalent between lib and ext. I can update the footnote text in a different PR.

Suggestion:

```suggestion
   * - ext + lib 1.15 [#PHP1.15-version-parity]_
```

**Review by mongoKart — CHANGES_REQUESTED** [review]
Two small suggestions. Note that docs-ecosystem/master already contains v1.15 but not v8.2 of the Driver, so the merge can accept either version in the conflict.

File: source/includes/language-compatibility-table-php.rst [hunk 2]
```diff
@@ -13,8 +14,8 @@
      - PHP 7.0
      - PHP 5.6
      - PHP 5.5
-
-   * - ext 1.15 + lib 1.15 [#PHP1.15-version-parity]_
+   * - ext + lib 1.15 [#PHP1.15-version-parity]_
```
Comment (jmikola) [inline source/includes/language-compatibility-table-php.rst:17]:
```suggestion

   * - ext + lib 1.15 [#PHP1.15-version-parity]_
```

File: source/includes/mongodb-compatibility-table-php.rst [hunk 3]
```diff
@@ -16,8 +16,7 @@
      - MongoDB 3.2
      - MongoDB 3.0
      - MongoDB 2.6
-  
-   * - ext 1.15 + lib 1.15 [#PHP1.15-version-parity]_
+   * - ext + lib 1.15 [#PHP1.15-version-parity]_
```
Comment (jmikola) [inline source/includes/mongodb-compatibility-table-php.rst:19]:
```suggestion

   * - ext + lib 1.15 [#PHP1.15-version-parity]_
```

**Pull request merged**